docs/raw/organisation-channels.md

```diff
@@ -0,0 +1,160 @@
+---
+permalink: /spec/17
+parent: Stable specs
+title: 17/ORGANISATION-CHANNELS
+---
+
+# 17/ORGANISATION-CHANNELS
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+>
+
+- [Organisation channels](#17organisation-channels)
+  - [Reason](#reason)
+  - [Fetching information about an organisation](#fetching-information-about-an-organisation)
+  - [Joining an organisation](#joining-an-organisation)
+  - [Updating channels](#updating-channels)
+
+## Reason
+
+Organisation channels are a way to allow users to build communities and create curated content.
+
+## Fetching information about an organisation
+
+An organisation is uniquely identified by a public key.
+
+This public key can be discovered through different means, links, indexable web pages or search directories for example.
+
+Given that the public key of the organisation is known, information about the channels and members can be fetched by listening to a discovery topic derived by the public key:
+
+```
+   Does not matter which topic for now, as long as it does not clash with public chats.
+```
+
+The organisation SHOULD publish periodically an `OrganisationDescription` message,
+at least once every 24 hours, ideally more to minimise the dependency on mailservers.
+
+
+```protobuf
+message ChatMessageIdentity {
+  // stubbing this, it's already been prepared in a different PR
+  string ens_name = 1;
+  string profile_picture = 2; // not a string, more complex data structure, but you get the point
+  string display_name = 3; 
+  string description = 4; // this is actually unique to an organisation, and chatMessageIdentity might want to be updated, or similar
+
+}
+message OrganisationPermissions {
+  enum Access {
+    UNKNOWN_ACCESS = 0;
+    NO_MEMBERSHIP = 1;
```

Comment:
To clarify, this refers to 'No membership required to Write', correct?

Reply:
correct, same as current public chats essentially